modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -1,25 +1,29 @@
 # frozen_string_literal: true
 
-require 'evss/error_middleware'
-
 module ClaimsApi
   module V2
     module Veterans
       class ClaimsController < ClaimsApi::V2::ApplicationController
-        # TODO: REMOVE BEFORE IMPLEMENTATION
-        skip_before_action :authenticate, only: %i[index]
-        ICN_FOR_TEST_USER = '1012667145V762142'
-        # TODO: REMOVE BEFORE IMPLEMENTATION
-
         def index
-          raise ::Common::Exceptions::Unauthorized if request.headers['Authorization'].blank?
-          unless params[:veteranId] == ICN_FOR_TEST_USER
-            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Resource not found')
-          end
+          raise ::Common::Exceptions::Forbidden unless user_is_target_veteran? || user_is_representative?
+
+          service           = bgs_service(veteran_participant_id: target_veteran.participant_id)
+          service_params    = { participant_id: target_veteran.participant_id }
+          bgs_claims        = service.benefit_claims.find_claims_details_by_participant_id(service_params)
+
+          query_params      = { veteran_icn: target_veteran.mpi.icn, status: %w[pending errored] }
```

Comment:
why just pending or errored here? we should get established claims as well so we can respond with our guid instead of the upstream id

Reply:
Hmmmm, maybe I'm misinterpreting the acceptance criteria?
> "Should return both our pending/errored claims and established claims"

I read that as "return our (Lighthouse) claims that have a status of 'pending'/'errored' and return claims from BGS".

Reply:
ah, gotcha. the blurb for the value in the id field is:
> id:           GUID from our system if we know about the claim, evss_id from EVSS if we do not have a record for that claim
which is for established claims as well though, any claim we know about, whatever state its in

Reply:
[tweaked here](https://github.com/department-of-veterans-affairs/vets-api/pull/7395/commits/a3468432550dba6ce15206eb20b5251ed13d7112)